pkg/controller/constraint/constants.go

```diff
@@ -0,0 +1,12 @@
+package constraint
+
+const (
+	// VapGenerationLabel indicates opting in and out preference for generating VAP objects.
+	VapGenerationLabel = "gatekeeper.sh/use-vap"
+	// VapFlagNone:do not generate.
```

Comment:
nit: can we align the comments? something like

```suggestion
	// VapFlagNone indicates opting out of generating VAP objects
```

Reply:
IMO we should keep the comments as is since the label one describes what the label is used for and others are the values one can set for the label and describes what each value means.